css-positioning/css-positioning/relative.css

```diff
@@ -0,0 +1,43 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+  position: relative;
+  top: 10px;
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 100%;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 26px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 65px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 137px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 350px;
+}
```

Comment:
You can remove `position: relative` from each of these, since you set it above in the `.pokemon > img` rule